engine/src/main/scala/cromwell/services/ServiceRegistryActor.scala

```diff
@@ -40,7 +40,7 @@ case class ServiceRegistryActor(globalConfig: Config) extends Actor {
   import ServiceRegistryActor._
 
   val services: Map[String, ActorRef] = serviceNameToPropsMap(globalConfig) map {
-    case (name, props) => name -> context.actorOf(props, "ServiceRegistryActor")
```

Comment:
This did not handle multiple services, as all were getting instantiated with the same name. This is one of the WIP stuff, I'll name each of the actors according to their name as defined in the config.